wandb/sdk/data_types/utils.py

```diff
@@ -182,12 +182,21 @@ def _log_table_artifact(val: "ValToJsonType", key: str, run: "LocalRun") -> None
         key: The key used to log val.
         run: The LocalRun used to log val.
     """
+    # Ensure that val is a WBValue before continuing
+    if not isinstance(val, WBValue):
```

Comment:
The type annotation on the function only requires `val` to be `ValToJsonType`, so this check seems wrong.